cmd/state/main.go

```diff
@@ -246,6 +246,10 @@ func run(args []string, isInteractive bool, cfg *config.Instance, out output.Out
 		if childCmd != nil {
 			cmdName = childCmd.UseFull() + " "
 		}
+		nonInteractiveError := &prompt.NonInteractiveError{}
+		if errors.As(err, &nonInteractiveError) {
```

Comment:
Is there a reason to assign the error struct to a variable here?

Reply:
Yes. Go wouldn't compile without it. I didn't want to use the variable :(